src/python/pants/backend/jvm/tasks/jvm_compile/jvm_classpath_publisher.py

```diff
@@ -17,6 +17,8 @@ def register_options(cls, register):
     super().register_options(register)
     register('--manifest-jar-only', type=bool, default=False,
              help='Only export classpath in a manifest jar.')
+    register('--internal-classpath-only', type=bool, default=True,
+             help='Only export the classpath of source targets, not 3rdparty jars.')
```

Comment:
I know third party libraries is typically what binary jar dependencies are typically used for, but is there a more precise term you can use here? Perhaps `jar_library` dependencies?

Reply:
Done!